test/e2e-tests.sh

```diff
@@ -50,6 +60,8 @@ set +o pipefail
 kubectl delete --ignore-not-found=true builds --all
 kubectl delete --ignore-not-found=true buildtemplates --all
 
+wait_for_running_build_pods
```

Comment:
I think that `wait_until_pods_running knative-build` would achieve the same thing with less code.

Interesting, looks like the script never really checked for the pods being up, like serving or eventing do (or maybe it was lost in a refactoring?).

Reply:
Oh thanks for pointing that out - i'll make the changes

Reply:
Funny, I don't see the changes here. Was it intentional?